downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -291,10 +291,77 @@ abstract class VersionControlSystem {
      *
      * @throws DownloadException In case the download failed.
      */
-    abstract fun download(
+    open fun download(
         pkg: Package, targetDir: File, allowMovingRevisions: Boolean = false,
         recursive: Boolean = true
```

Comment:
You could take the chance and properly format this with one argument per line.

Reply:
Done.